src/Sylius/Behat/Context/Setup/PaymentContext.php

```diff
@@ -82,20 +82,21 @@ public function __construct(
     }
 
     /**
-     * @Given the store allows paying :paymentMethodName
-     * @Given the store allows paying with :paymentMethodName
+     * @Given the store (also) allows paying :paymentMethodName
+     * @Given the store (also) allows paying with :paymentMethodName
```

Comment:
Probably not, but it is old step. I can fix it here if it won't cause to many problems.